languageserver/server/server.go

```diff
@@ -873,7 +873,7 @@ var containerCompletionItems = []*protocol.CompletionItem{
 // Completion is called to compute completion items at a given cursor position.
 //
 func (s *Server) Completion(
-	_ protocol.Conn,
+	conn protocol.Conn,
```

Comment:
Probably doesn't need this change?

Reply:
Good catch! I used it to log something client-side, but yeah, not needed anymore 👍